bindings/python/tests/implementations/test_sentencepiece.py

```diff
@@ -14,10 +16,40 @@ def test_train_from_iterator(self):
 
 
 class TestSentencePieceUnigram:
+    def test_train(self, tmpdir):
+        p = tmpdir.mkdir("tmpdir").join("file.txt")
+        p.write("A first sentence\nAnother sentence\nAnd a last one")
+
+        tokenizer = SentencePieceUnigramTokenizer()
+        tokenizer.train(files=str(p), show_progress=False)
+
+        output = tokenizer.encode("A sentence")
+        assert output.tokens == ["▁A", "▁", "s", "en", "t", "en", "c", "e"]
```

Comment:
Could you assert it triggers an exception with unknown token ?

Reply:
Great idea! I'll do it :slightly_smiling_face: